### DISCUSSION TITLE
Start plan using command line

### DISCUSSION THREAD

**Discussion by ChrisB85** [garethgeorge/backrest#275] [Q&A] [answered]
Is there a way to run specific plan using command line? I would like to run one of my plans after DB dump is complete. Is it possible?

**Comment by garethgeorge** [comment, marked as answer]
You can do this with curl (with the caveat that I don't guarantee this API will stay stable)

```
curl -X POST 'localhost:9898/v1.Backrest/Backup' --data '{"value": "YOUR_PLAN_ID"}' -H 'Content-Type: application/json'
```

note: this will only work if you disable authentication.

**Reply by gfordinal** [reply]
Although this works, it feels not right to disable auth, just to be able to run a scripted backup through the API.
